mne/viz/_brain/tests/test.ipynb

```diff
@@ -39,16 +42,40 @@
     "initial_time = 0.13\n",
     "mne.viz.set_3d_backend('notebook')\n",
     "brain_class = mne.viz.get_brain_class()\n",
-    "for interactive_state in (False, True):\n",
-    "    plt.interactive(interactive_state)\n",
+    "\n",
+    "\n",
+    "@contextmanager\n",
+    "def interactive(on):\n",
+    "    old = plt.isinteractive()\n",
+    "    plt.interactive(on)\n",
+    "    try:\n",
+    "        yield\n",
+    "    finally:\n",
+    "        plt.interactive(old)\n",
+    "\n",
+    "with interactive(False):\n",
     "    brain = stc.plot(subjects_dir=subjects_dir, initial_time=initial_time,\n",
     "                     clim=dict(kind='value', pos_lims=[3, 6, 9]),\n",
     "                     time_viewer=True,\n",
-    "                     hemi='split')\n",
+    "                     show_traces=True,\n",
+    "                     hemi='split', size=300)\n",
```

Comment:
can we do `hemi='lh'` to reduce test time, or is the split-ness necessary / good to check?

Reply:
Should be fine with `hemi=lh`, I'll push that right now

Reply:
5d34669:

```
12.76s call     mne/viz/_brain/tests/test_notebook.py::test_notebook_3d_backend
```

1e28b73:
```
8.68s call     mne/viz/_brain/tests/test_notebook.py::test_notebook_3d_backend
```

And 30s speedup overall.